services/masterdata/src/lib/components/masterdata-type-edit-form/masterdata-type-edit-form.component.ts

```diff
@@ -136,4 +140,15 @@ export class MasterdataTypeEditFormComponent implements OnInit {
 	navigateToListView() {
 		this.router.navigate(["/"]);
 	}
+
+	private buildTranslationTexts() {
+		this.languageService.onTranslationsLoaded(() => {
```

Comment:
Tip: to make this code more compact you can also prepare the right resource key and then get if from the language service:


const mode = this.isAddForm ? "add" : "edit";
const headerKey = `masterdatamgmt.pages.masterdataTypeUpsert.${mode}FormTitle`
this.formHeaderText = this.translateService.instant(headerKey)